src/CMakeLists.txt

```diff
@@ -30,7 +30,13 @@ if(USE_FONTS)
 endif()
 
 if(WIN32)
-  list(APPEND mudlet_RCCS icons/mudlet_main_512x512_6XS_icon.ico)
+  if(BUILD matches "-ptb.+")
+    list(APPEND mudlet_RCCS icons/mudlet_ptb.ico)
+  elseif((BUILD matches "-ptb.+") or (BUILD matches "-test.+"))
```

Comment:
For some reason, this checks for "ptb" twice?

Reply:
Ah, yeah the first should have been `"-dev.+"`...

Reply:
Fixed now.